LocalPayment/src/androidTest/java/com/braintreepayments/api/LocalPaymentClientTest.java

```diff
@@ -7,6 +7,10 @@
 import androidx.test.core.app.ApplicationProvider;
 import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner;
 
+import com.braintreepayments.api.localpayment.LocalPaymentAuthRequest;
```

Comment:
these imports should be unnecessary as they are local to the package once tests are also moved